src/components/Events/Fall_GM/Fall_GM.js

```diff
@@ -5,57 +5,196 @@ import Countdown from 'react-countdown';
 
 export default class Fall_GM extends React.Component {
 	render() {
-		const FALL_GM_2020_START_TIME = new Date('2020-10-05T18:30:00-0700'); //October 5th 2020 at 6:30 PDT 
-	
-		const renderHumanReadableTime = ({ days, hours, minutes, seconds, completed }) => {
-			if (completed) return <h1 id="time-to-gm-headline-now">Fall General Meeting 2020 is happening right now!</h1>; //EVAN TODO: add the rest of the program and links to zoom / slack here
-	
-			let dayString ="Day";
-			let hourString = "Hour";
-			let minuteString = "Minute";
-			let secondString = "Second";
-	
-			if (days !== 1) dayString += 's';
-			if (hours !== 1) hourString += 's';
-			if (minutes !== 1) minuteString += 's';
-			if (seconds !== 1) secondString += 's';
+		const FALL_GM_2020_START_TIME = new Date('2020-10-05T18:30:00-0700'); //October 5th 2020 at 6:30 PDT
+
+		const renderCountdownOrNow = ({ days, hours, minutes, seconds, completed }) => {
+			if (completed) return <GM_NOW/>;
+			else {
+				let dayString = "Day";
+				let hourString = "Hour";
+				let minuteString = "Minute";
+				let secondString = "Second";
+				if(days !== 1) dayString += "s";
+				if(hours !== 1) hourString += "s";
+				if(minutes !== 1) minuteString += "s";
+				if(seconds !== 1) secondString += "s";
```

Comment:
Two quick things here, neither of which are merge-blocking:

1. AFAIK it's somewhat not-great practice to use one-line if statements without braces; makes things a little hard to read and slightly error-prone. I do think the semicolon makes it a little less ambiguous.
2. You can rewrite this using string comprehension, though it may reduce clarity - your call

```js
let dayString = `Day${days !== 1 ? "s" : ""}`;
...
```